src/reference/config/bindings/asyncapi/.partials/proxy.yaml

```
@@ -0,0 +1,37 @@
+  asyncapi_proxy:
+    type: asyncapi
+    kind: proxy
+    options:
+      specs:
+        my-mqtt-api-spec:
+          my-kafka-api-spec:
+            catalog:
+              my_catalog:
+                subject: petstore
+                version: latest
+        my-kafka-api-spec:
+          my-kafka-api-spec:
+            catalog:
+              my_catalog:
+                subject: petstore
+                version: latest
```

```suggestion
        my-mqtt-api-spec:
          catalog:
            my_catalog:
              subject: mqtt-streetlights
              version: latest
        my-kafka-api-spec:
          catalog:
            my_catalog:
              subject: kafka-streetlights
              version: latest
```
This suggestion changes the syntax (I think it was not schema compliant), and the subjects to be based on standard AsyncAPI example specifications, i.e streetlights.